tools/etcd-backup/README.md

```diff
@@ -0,0 +1,57 @@
+# Etcd Backup
+
+## Overview
+
+The etcd-backup trigger a backup process of etcd-cluster via etcd-backup-operator.
```

Comment:
The etcd-backup triggers a backup process of etcd-cluster using the etcd-backup-operator.